shared/templates/pam_account_password_unix_remember/oval.template

```diff
@@ -1,57 +1,59 @@
-{{% if product in [ "sle12", "sle15" ] %}}
+{{% if product in [ "sle12", "sle15" ] or "debian" in product or "ubuntu" in product %}}
 {{%- set accounts_password_pam_unix_remember_file = '/etc/pam.d/common-password' -%}}
 {{% else %}}
 {{%- set accounts_password_pam_unix_remember_file = '/etc/pam.d/system-auth' -%}}
 {{% endif %}}
 
 <def-group>
-  <definition class="compliance" id="{{{ rule_id }}}" version="2">
+  <definition class="compliance" id="accounts_password_pam_unix_remember" version="2">
     {{{ oval_metadata("The passwords to remember should be set correctly.") }}}
     <criteria operator="OR" comment="Check if pam_pwhistory.so or pam_unix.so are configured">
       <criteria operator="AND" comment="Check if pam_pwhistory.so is properly configured">
         <!--
-          pam_pwhistory.so parameters can be defined directly in pam files or, in newer versions,
-          in /etc/security/pwhistory.conf. The last is the recommended option when available. Also,
-          is the option used by auselect tool. However, regardless the approach, a minimal
-          declaration is common in pam files. -->
+            pam_pwhistory.so parameters can be defined directly in pam files or, in newer versions,
+            in /etc/security/pwhistory.conf. The last is the recommended option when available. Also,
+            is the option used by auselect tool. However, regardless the approach, a minimal
+            declaration is common in pam files. -->
         <criterion test_ref="test_accounts_password_pam_unix_remember"
-          comment="pam_pwhistory.so is properly defined in password section of PAM file"/>
+		   comment="pam_pwhistory.so is properly defined in password section of PAM file"/>
```

Comment:
@a-skr like here in this file
There's a lot of indentation changes.

Reply:
I see...

I mixed content from both ubuntu and shared oval files, but they don't use the same indentation. I will fix it with next PR update.